### DISCUSSION TITLE
'i2s_read_bytes'

### DISCUSSION THREAD

**Discussion by KingSalamander216** [community/community#70082] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Good day! after compiling there's an error:

'i2s_read_bytes' was not declared in this scope.

I really want to know how to make correct it and make it work. I would really appreciate your help.

**Comment by Dineth-H** [comment]
The error message "i2s_read_bytes was not declared in this scope" means that the compiler cannot find the function i2s_read_bytes() in the current scope. This can happen for a few reasons:

- You have not included the header file that declares the function.

- You have misspelled the function name.

- The function is in a different namespace than the code where you are calling it.

To fix this error, 
you need to make sure that the **function is declared** in the current scope.
This can be done by including the header file that declares the function, or by using the using keyword to import the namespace where the function is declared.

Once you have declared the i2s_read_bytes() function, you should be able to compile and run your code without any errors.

- [ ] Make sure that you have installed the correct libraries.
- [ ] Try cleaning and rebuilding your project.
